Started by upstream project "[8mha:////4N+xKAmskkIjiAKi7Ga0bQ6psYNpIm20xSKMddamLXXhAAAAnh+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAzeEgZh/az8JH3fRKD2ioCi/KzU5BJ9ACRKKtTIAAAA[0mMatrixProject" build number [8mha:////4Dx9D8nIr8PVapKWHgtPRfydpOsk68peTDycO96Lz2WtAAAAoB+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAymEgZR/az8JH3fRKD2ioCi/KzU5BJ9EwMAOXlhK8oAAAA=[0m40
originally caused by:
 Started by user [8mha:////4BZv/RbSRYtMZa6Gy4V7aSsv1Lgf4frwfCHza6l7zHyjAAAAlx+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAzWEgZu/dLi1CL9xJTczDwAj6GcLcAAAAA=[0madmin
Building remotely on [8mha:////4HoEpZdikjvqR9+5tuPMTRxW5qn7y/sTx7ped+3YssuvAAAAnB+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAy2EgYB/eT83ILSktQi/eKcxLJUIwCCp2lrxQAAAA==[0mslave2 (label2) in workspace C:\Jenkins\workspace\MatrixProject\label\slave2
 > git.exe rev-parse --is-inside-work-tree # timeout=10
Fetching changes from the remote Git repository
 > git.exe config remote.origin.url https://github.com/OlegGlizerin/TeamRepo.git # timeout=10
Fetching upstream changes from https://github.com/OlegGlizerin/TeamRepo.git
 > git.exe --version # timeout=10
 > git.exe fetch --tags --progress https://github.com/OlegGlizerin/TeamRepo.git +refs/heads/*:refs/remotes/origin/*
Checking out Revision fb6100a51c82ef5b477b7389c077b0f0fc486f22 (refs/remotes/origin/oleg)
 > git.exe config core.sparsecheckout # timeout=10
 > git.exe checkout -f fb6100a51c82ef5b477b7389c077b0f0fc486f22
Commit message: "test without datorama"
 > git.exe rev-list fb6100a51c82ef5b477b7389c077b0f0fc486f22 # timeout=10
[slave2] $ cmd.exe /C "c:\Python27\python.exe C:\Users\JACKSP~1\AppData\Local\Temp\shiningpanda1245487890907312795.py && exit %%ERRORLEVEL%%"
Traceback (most recent call last):
  File "C:\Users\JACKSP~1\AppData\Local\Temp\shiningpanda1245487890907312795.py", line 3, in <module>
    projectBaseDir = os.path.dirname(sys.modules['__main__'].__file__)
NameError: name 'sys' is not defined
Build step 'Custom Python Builder' marked build as failure
Notifying upstream projects of job completion
Finished: FAILURE
